kitsune/customercare/zendesk.py

```diff
@@ -101,8 +101,8 @@ def create_ticket(self, user, ticket_fields):
                 ]
             )
         ticket = Ticket(
-            subject=ticket_fields.get("subject") or ticket_fields.get("category"),
-            comment={"body": ticket_fields.get("description") or ticket_fields.get("category")},
+            subject=ticket_fields.get("subject") or f"{product_config['name']} support",
+            comment={"body": ticket_fields.get("description") or "No response provided."},
```

Comment:
Let's add the `No response provided` in a var at the beginning of the file

Reply:
Can do.